core/unit_test/TestExecSpaceThreadSafety.hpp

```diff
@@ -78,9 +78,11 @@ void run_exec_space_thread_safety_range() {
 
 TEST(TEST_CATEGORY, exec_space_thread_safety_range) {
 #ifdef KOKKOS_ENABLE_OPENACC  // FIXME_OPENACC
+#ifdef KOKKOS_ENABLE_OPENACC_FORCE_HOST_AS_DEVICE
```

Comment:
Please confirm that this means that the test works EXCEPT when targeting the host with the OpenACC backend.
Meaning we could have gotten of these guards if it wasn't for that new option we introduce.

Reply:
Yes, the tests work except when the OpenACC backend targets the host.